std/stdio.d

```diff
@@ -3326,7 +3326,21 @@ void main()
         assert(dcharsOut == "foo");
     }
 
-/// Get the size of the file, ulong.max if file is not searchable, but still throws if an actual error occurs.
+/** Get the size of the file in bytes, ulong.max if file is not searchable, but still throws if an actual error occurs.
```

Comment:
Get_s_ the size of the file in bytes, ulong.max if file is not searchable, ~~but still~~ _or_  throws if an ~~actual error occurs~~ the operation fails.

Reply:
Incorporated your suggestion and then some